pkgs/by-name/vo/vodozemac-bindings-cpp/package.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  cargo,
+  perl,
+  rustPlatform,
+  rustc,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "vodozemac-bindings-cpp";
+  version = "0.2.0";
+
+  src = fetchFromGitLab {
+    domain = "lily-is.land";
+    owner = "kazv";
+    repo = "vodozemac-bindings";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-DOc5GxMgpecJeQtkDSwvGD+bOyCZ8XRQ/uC/RAnKQv4=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoVendor {
+    inherit (finalAttrs) src;
```

```suggestion
    inherit (finalAttrs) pname version src;
```

Forcing people to not forget about updating hash. Hash will change after this